README.md

```diff
@@ -0,0 +1,150 @@
+# NimbleHex
+
+NimbleHex allows self-hosting of Hex packages.
+
+Features:
+
+* Pluggable storage. NimbleHex ships with local filesystem and S3 implementations
+* Publishing packages via HTTP API
+* Support for multiple repositories
+* Mirroring
+
+## Setup
```

Comment:
I also need to write a section about setting the expectations of the project, for people to actually use it they'd need to configure backups, ssl, auth, and more, and that's out of the scope of this project.